sys/openbsd/dev_vmm.txt

```diff
@@ -41,14 +43,8 @@ vcpu_segment_info {
 vm_create_params {
 	vcp_nmemranges	len[vcp_memranges, int64]
 	vcp_ncpus	int64
-	vcp_ndisks	len[vcp_disks, int64]
-	vcp_nnics	int64
```

Comment:
I would comment these out with some explanatory comment.
Or add defines for these consts with some values and a comment that they are wrong.
These fields are present in the actual kernel interfaces, right?
We just need some additional includes, right?

But also this int64 field  vcp_nnics does not depend on any consts, why is it removed?

@blackgnezdo

Reply:
It addressed changes from https://github.com/openbsd/src/commit/73a98491f956970e5755ae565866cd809697967e#diff-c679d226df45db32b92778029c243d83c9f77bd6fad39d322b86705c74e3f267L472